### PR TITLE
Replace home page banner image

### PR DESCRIPTION
**Pull request by gilgongo**
This was [briefly suggested with mild agreement](https://github.com/jamulussoftware/jamuluswebsite/issues/841#issuecomment-1296349139) a while ago: to show a screenshot of Jamulus on the home page so as to give people a better idea of what to expect.

**Does this need translation?**

YES

(since the screenshot could be in other languages)

----

image

----

iPhone 12 Pro:

image

----

image

## Checklist

- [ ] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/main/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [ ] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [x] I'm sure that this Pull Request goes to the correct branch

### CODE REVIEW

**Comment by pljones** [comment]
Could you also post both landscape and portrait mobile browser screenshots, please?

**Comment by pljones** [comment]
@jamulussoftware/designers @jamulussoftware/translators @jamulussoftware/maindevelopers 
Are we okay to merge this?  If so, please approve.

File: assets/css/home.css [hunk 1]
```diff
@@ -81,3 +81,21 @@ body.is-os-other #quick_dl_container a.os-other {
   transition: 0.4s;
   opacity: 0.9;
 }
+
+figure {
```
Comment (ann0see) [inline assets/css/home.css]: I'd prefer a separate class for this image instead of styling the whole figure.

Comment (gilgongo) [reply assets/css/home.css]: Note sure I know how to do that properly. Can you give an example?

Comment (ann0see) [reply assets/css/home.css]: For the figure in question add a class attribute: `class="sth"` then use the sth class in css

Comment (gilgongo) [reply assets/css/home.css]: To have the style for the figure part of the class, you mean? Would we not do that for all figures though? I didn't want to change too much with this PR (there is also https://github.com/jamulussoftware/jamuluswebsite/issues/673 BTW which may be related)

Comment (ann0see) [reply assets/css/home.css]: I mean if you think that it's for all figures put it into fw.css not home.css. 

I'd just assume that you may not want the exact same styling for each figure?
Maybe the center alignment is different? If you're sure that you want to apply this for each figure element, then go for it. But not in home.css

File: wiki/en/misc/1-index.md [hunk 2]
```diff
@@ -27,9 +27,12 @@ Jamulus lets you play, rehearse, or jam with your friends, your band, or anyone
 </div>
 <div class="fx-row fx-row-center-xs" id="bannercontainer">
   <div class="fx-col-100-xs">
-    <a href="wiki/Getting-Started">
-      <img alt="Jamulus Banner. Links to getting started page" src="{{ '/assets/img/jamulusbannersmall.png' | relative_url }}" id="jamulusbanner" loading="lazy" />
-    </a>
+    <figure>
+      <a href="wiki/Getting-Started" rel="canonical">
+      <img src="{% include img/en-screenshots/main-screen-medium.inc %}" style="border: 5px solid grey;" id="jamulusbanner" loading="lazy" alt="A screenshot of the main mixer window showing five people from different countries connected.">
```
Comment (ann0see) [inline wiki/en/misc/1-index.md:32]: Also add the solid gray border thing either in a separate class or add figure>img rule in the css

File: assets/css/home.css (hunk 1 above)
Comment (gilgongo) [reply assets/css/home.css]: I didn't intend this to be a global figure style change, but I'm also not sure why the figure isn't styled the same as the others unless I duplicate the the figure in home.css. Anyway, I'll change that so it just has a class to center the text.

File: assets/css/home.css [hunk 3]
```diff
@@ -81,3 +81,7 @@ body.is-os-other #quick_dl_container a.os-other {
   transition: 0.4s;
   opacity: 0.9;
 }
+
+.sth {
```
Comment (ann0see) [inline assets/css/home.css]:
```suggestion
.mainbannerfig {
```

File: wiki/en/misc/1-index.md [hunk 4]
```diff
@@ -27,9 +27,12 @@ Jamulus lets you play, rehearse, or jam with your friends, your band, or anyone
 </div>
 <div class="fx-row fx-row-center-xs" id="bannercontainer">
   <div class="fx-col-100-xs">
-    <a href="wiki/Getting-Started">
-      <img alt="Jamulus Banner. Links to getting started page" src="{{ '/assets/img/jamulusbannersmall.png' | relative_url }}" id="jamulusbanner" loading="lazy" />
-    </a>
+    <figure class="sth">
```
Comment (ann0see) [inline wiki/en/misc/1-index.md]:
```suggestion
    <figure class="mainbannerfig">
```

**Review by ann0see — COMMENTED** [review]
The naming should be a bit more expressive...

File: assets/css/home.css (hunk 1 above)
Comment (ann0see) [reply assets/css/home.css]: Probably this has to do with the order in which the css files get loaded. home.css is strictly only for what happens on the home page while fw.css is integrated everywhere.

**Review by ann0see — APPROVED** [review]
Still the inline css is a bit questionable. Approving anyway.

**Pull request merged**